src/main/kotlin/no/nav/tiltakspenger/soknad/api/pdl/Person.kt

```diff
@@ -38,6 +38,11 @@ data class Person(
             .mapNotNull { it.relatertPersonsIdent }
             .distinct()
     }
+
+    fun erUnder16År(): Boolean {
+        val datoFor16ÅrSiden = LocalDate.now().minusYears(16)
+        return this.fødselsdato!!.isAfter(datoFor16ÅrSiden)
```

Comment:
Kanskje unngå double bang? Feks:

this.fødselsdato?.let {
    return it.isAfter(datoFor16ÅrSiden)
}
throw IllegalStateException("Barn mangler fødselsdato")